frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -140,6 +140,42 @@ function filemanagerutil.genResetSettingsButton(file, caller_callback, button_di
     }
 end
 
+function filemanagerutil.genBookInformationButton(file, dialog, button_disabled)
+    return {
+        text = _("Book information"),
+        id = "book_information", -- used by covermenu
+        enabled = not button_disabled,
+        callback = function()
+            UIManager:close(dialog)
+            require("apps/filemanager/filemanagerbookinfo"):show(file)
```

Comment:
I guess these require() are to avoid require loops ?
Any reason to want these 3 filemanagerutil.genBookSomethingButton in filemanagerutil, and not in filemanagerbookinfo (as they are related to bookinfo) ?

Reply:
Yes, to avoid loops.
To keep all buttons in one place.

Reply:
Oh, ok, there were already 3 xyzButton function in filemanagerutil.